man/systemd.netdev.xml

```diff
@@ -432,6 +432,24 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>EgressQOSMaps=</varname></term>
+        <listitem>
+          <para>The EgressQOSMaps defines a mapping of Linux internal packet priority (SO_PRORITY) to VLAN header
+          PCP field for outgoing frames. A whitespace-separated list of unsigned integers in the format from-to i.e.
+          21:15 45:5. When unset, the kernel's default will be used.
```

Comment:
Please use `<literal>` tag for the example. Also mention that `from` must be larger than or equal to `to`.